### PR TITLE
[webui] add mailcatcher gem for testing emails

### PR DESCRIPTION
**Pull request by evanrolfe**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@evanrolfe with [letter_opener](https://github.com/ryanb/letter_opener) and [letter_opener_web](https://github.com/fgrehm/letter_opener_web) you don't need to open ports.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -139,5 +139,12 @@ You can access the frontend at [localhost:3000](http://localhost:3000). Whatever
     ```
     vagrant ssh
     ```
+13. Changed an email? Test it with mailcatcher:
```
Comment (hennevogel) [inline CONTRIBUTING.md]: nitpick: I'm not sure this need to be in the contribution guide. It's already pretty complicated...

File: Vagrantfile [hunk 2]
```diff
@@ -27,8 +27,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     fe.exec.commands 'script/start_test_backend', directory: '/vagrant/src/api'
     fe.exec.commands 'contrib/start_development_backend', directory: '/vagrant'
     fe.exec.commands '*', directory: '/vagrant'
+    fe.exec.commands 'script/start_mailcatcher', directory: '/vagrant/src/api'
```
Comment (hennevogel) [inline Vagrantfile]: Why don't you start this always with foreman?

**Comment by hennevogel** [comment]
letter opener is also slightly better maintained according to [letter_opener](https://ossert.evilmartians.io/letter_opener) vs. [mailcatcher](https://ossert.evilmartians.io/mailcatcher) on the awesome ossert service!

**Comment by evanrolfe** [comment]
Ok i will close this PR and add letter_opener later.

**Pull request closed without merging**